packages/office-ui-fabric-react/src/components/Facepile/Facepile.test.tsx

```diff
@@ -0,0 +1,133 @@
+          overflowButtonProps={ {} }
+          overflowButtonType={ OverflowButtonType.more }
+          />);
+      let overflowButton = wrapper.find('.ms-Icon--More');
+      expect(overflowButton).to.have.length(1, 'Overflow button should render');
+      let buttons = wrapper.find('.ms-Facepile-itemButton');
+      expect(buttons).to.have.length(1, 'Only one button should be rendered');
+    });
+
+    it('renders without descriptive overflow button if overflowButtonProps are not null and maximum personas are not exceeded', () => {
+      const wrapper = shallow(
+        <Facepile personas={ [] }
+          overflowButtonProps={ {} }
+          overflowButtonType={ OverflowButtonType.descriptive }
+          />);
+      let overflowButton = wrapper.find('.ms-Facepile-descriptiveOverflowButton');
+      expect(overflowButton).to.have.length(0, 'Overflow button should not render');
+      let buttons = wrapper.find('.ms-Facepile-itemButton');
+      expect(buttons).to.have.length(0, 'No buttons should be rendered');
+    });
+
+    it('renders with descriptive overflow button if overflowButtonProps are not null and maximum personas are exceeded', () => {
+      let personas: IFacepilePersona[] = facepilePersonas.concat(...facepilePersonas, ...facepilePersonas);
+      const wrapper = shallow(
+        <Facepile
+          personas={ personas }
+          maxDisplayablePersonas={ 5 }
+          overflowButtonProps={ {} }
+          overflowButtonType={ OverflowButtonType.descriptive }
+          />);
+      let overflowButton = wrapper.find('.ms-Facepile-descriptiveOverflowButton');
+      expect(overflowButton).to.have.length(1, 'Overflow button should render');
+      let buttons = wrapper.find('.ms-Facepile-itemButton');
+      expect(buttons).to.have.length(6, 'All six personas and overflow button should be rendered');
+    });
+
+    it('renders no more than maximum allowed personas', () => {
+      const wrapper = shallow(
+        <Facepile
+          personas={ facepilePersonas.concat(facepilePersonas, facepilePersonas, facepilePersonas) }
+          maxDisplayablePersonas={ 2 }
+          />);
+      let buttons = wrapper.find('.ms-Facepile-itemButton');
+      expect(buttons).to.have.length(2, 'Only two buttons should be rendered');
+    });
+
+    it('persona is clickable if onClick property is set', () => {
+      let clicked = 0;
+      let personas: IFacepilePersona[] = [{
+        personaName: 'Alex Lundberg',
+        onClick: (ev: React.MouseEvent<HTMLElement>, persona: IFacepilePersona) => {
+          clicked++;
+          ev.preventDefault();
+        }
+      }];
+      const wrapper = mount(
+        <Facepile
+          personas={ personas }
+          />);
+      let buttons = wrapper.find('.ms-Facepile-itemButton');
+      expect(buttons).to.have.length(1, 'Clickable Persona should render');
+      buttons.simulate('click');
+      expect(clicked).to.be.equal(1, 'Persona should have been clickable');
+    });
+  });
```

Comment:
The tests you wrote are great, but there's no test testing the persona sizes specifically (your change)